docs/docs/50-user-guide/60-reference-docs/40-expressions.md

```diff
@@ -315,6 +315,8 @@ The optional `freightOrigin` argument should be used when a `Stage` requests
 `Freight` from multiple origins (`Warehouse`s) and more than one can provide a
```

Comment:
Probably an idea to add references to [optional chaining](https://expr-lang.org/docs/language-definition#optional-chaining) and [nil coalescing](https://expr-lang.org/docs/language-definition#nil-coalescing) to this document to help people supercharge the `nil` returns.

Reply:
Will add a note in the docs